digitalocean_k8s_ubuntu_16.04_master.sh

```diff
@@ -5,26 +5,38 @@
 # ------------------------------------------------------------------------------------------------------------------------
 
 # Specify the Kubernetes version to use.
-KUBERNETES_VERSION="1.9.2"
+KUBERNETES_VERSION="1.11.0"
 KUBERNETES_CNI="0.6.0"
+DOCKER_VERSION="17.03"
 
 # Obtain Droplet IP addresses.
 HOSTNAME=$(curl -s http://169.254.169.254/metadata/v1/hostname)
 PRIVATEIP=$(curl -s http://169.254.169.254/metadata/v1/interfaces/private/0/ipv4/address)
 PUBLICIP=$(curl -s http://169.254.169.254/metadata/v1/interfaces/public/0/ipv4/address)
-echo $PRIVATEIP > /tmp/.ip
+echo "$PRIVATEIP" > /tmp/.ip
```

Comment:
About your question for `/tmp/.ip`, I don't think it's needed anymore. It has been used for VPN, however, VPN is deprecated and removed.

However, we need to check once again have we removed it from code before removing it here.